### DISCUSSION TITLE
Get specific version NDK

### DISCUSSION THREAD

**Discussion by Semsem-programmer** [android/ndk#2075] [Q&A]
Where I can **find NDK (r23b)**, to download it for unity?

**Comment by pirama-arumuga-nainar** [comment]
https://dl.google.com/android/repository/android-ndk-r23b-linux.zip

The [Unsupported Downloads](https://github.com/android/ndk/wiki/Unsupported-Downloads) page shows the link for r23c but the r23b is also still present.  (As the link implies, this is still unsupported though).
